index.html

```diff
@@ -0,0 +1,98 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Microverse Portfolio</title>
+  </head>
+  <body>
+    <header class="navbar">
+      <div class="navbar-wrapper">
+        <a class="navbar-logo" href="#">My Logo</a>
+        <nav>
+          <ul class="nav-links">
+            <li><a class="nav-link" href="#">Portfolio</a></li>
+            <li><a class="nav-link" href="#">About</a></li>
+            <li><a class="nav-link" href="#">Contact</a></li>
+            <li>
+              <a class="nav-link" href="#"
+                ><img src="images/mail.svg" alt="Mail Link"
+              /></a>
+            </li>
+          </ul>
+
+          <img
+            class="menu"
+            src="images/menu.svg"
+            alt="Menu"
+          />
+        </nav>
+      </div>
+    </header>
+    <section class="headline-section">
+      <div class="headline-section-wrapper">
+        <h1 class="headline-title">Hey There.<br />I’m Htet.</h1>
+        <p class="headline-professional">I'm a Software Developer</p>
+        <p class="headline-description">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+        <ul class="headline-links">
+          <li>
+            <a class="headline-link" href="#"
+              ><img
+                class="headline-link-img"
+                src="images/github.svg"
+                alt="GitHub Link"
+            /></a>
+          </li>
+          <li>
+            <a class="headline-link" href="#"
+              ><img
+                class="headline-link-img"
+                src="images/linkedin.svg"
+                alt="Linkedin Link"
+            /></a>
+          </li>
+          <li>
+            <a class="headline-link" href="#"
+              ><img
+                class="headline-link-img"
+                src="images/angelist.svg"
+                alt="Angelist Link"
+            /></a>
+          </li>
+          <li>
+            <a class="headline-link" href="#"
+              ><img
+                class="headline-link-img"
+                src="images/twitter.svg"
+                alt="Twitter Link"
+            /></a>
+          </li>
+          <li>
+            <a class="headline-link" href="#"
+              ><img
+                class="headline-link-img"
+                src="images/medium.svg"
+                alt="Medium Link"
+            /></a>
+          </li>
+        </ul>
+      </div>
+      <img
+        class="headline-illustration"
+        src="images/illustration.svg"
+        alt="illustration"
+      />
+    </section>
```

Comment:
- [ ] Kindly use this image as a background image and not an actual image in the markup of this document. The image is for aesthetic reasons and does not convey important information regarding the data. Kindly use it as a background image. 👍 😎 

#### Hint: Use the background image for this section's container and then position it properly 💻